### DISCUSSION TITLE
WebTransport via tRPC

### DISCUSSION THREAD

**Discussion by konstantikkov** [trpc/trpc#4952] [Q&A]
Is there any options to use WebTransport for subscriptions with tRPC client/server like WebSockets?

**Comment by JonParton** [comment]
Interesting!

Just to share as I was unaware this is a good but a [bit out of date artical on Webtransport](https://ably.com/blog/can-webtransport-replace-websockets) and also  [the docs of Socket.io on their implementation of web transport.](https://socket.io/get-started/webtransport#:~:text=In%20short%2C%20WebTransport%20is%20an,.github.io%2Fwebtransport%2F)

Seems like it is still not fully mature, with things like  [safari not implementing yet](https://caniuse.com/webtransport) but looks to be an interesting future web sockets alternative!

**Comment by KATT** [comment]
No, but should be possible to make an adapter. We're working on SSE fwiw
